### PR TITLE
Link to local volume operations guide in sig-storage-local-static-provisioner

### PR DESCRIPTION
**Pull request by cofyc**
fixes https://github.com/pingcap/tidb-operator/issues/410
Operations on local volume setup are common in most environments, we can link to operation guides in `sig-storage-local-static-provisioner` project and maintain docs there. 

https://github.com/cofyc/tidb-operator/blob/fix410/docs/setup.md

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -63,47 +63,11 @@ You can follow Helm official [documentation](https://helm.sh) to install Helm in
 
 ## Local Persistent Volume
 
-Local disks are recommended to be formatted as ext4 filesystem. The local persistent volume directory must be a mount point: a whole disk mount or a [bind mount](https://unix.stackexchange.com/questions/198590/what-is-a-bind-mount):
+### Prepare local volumes
 
-### Disk mount
+See [operations guide in sig-storage-local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md) which contains of preparing, setting up and cleaning up local volumes on the nodes.
```
Comment (gregwebs) [inline docs/setup.md]:
```suggestion
See the [operations guide in sig-storage-local-static-provisioner](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/operations.md) which explains setup and cleanup of local volumes on the nodes.
```

File: docs/setup.md [hunk 2]
```diff
@@ -63,47 +63,11 @@ You can follow Helm official [documentation](https://helm.sh) to install Helm in
...
-Disk mount:
-```shell
-$ echo "/dev/nvme0n1 /mnt/disks/disk01 ext4 defaults 0 0" >> /etc/fstab
-```
-
-Bind mount:
-```shell
-$ echo "/data/local-pv01 /mnt/disks/local-pv01 none bind 0 0" >> /etc/fstab
-```
+It also provides some [best practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) in production.
```
Comment (gregwebs) [inline docs/setup.md]:
```suggestion
It also provides some [best practices](https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/docs/best-practices.md) for production.
```

**Comment by cofyc** [comment]
@gregwebs @aylei 
Thanks for reviewing this! I've committed suggestions. Can you approve again?

**Comment by weekface** [comment]
It is better to write the mount options to `/etc/fstab` and then use `mount -a` to mount them all?

**Comment by cofyc** [comment]
sure, `mount -a` can test `/etc/fstab` is valid or not

**Pull request merged**